src/status_im2/contexts/onboarding/profiles/style.cljs

```diff
@@ -58,3 +58,7 @@
   []
   {:margin-bottom (if platform/android? 20 46)})
 
+(def forget-password-doc-container {:margin-right 16})
+(def forget-password-step-container {:flex-direction :row :margin-top 14})
```

Comment:
nit, consider aligning the keys of this map

Reply:
You mean like this? Lint won't pass.
```clj
(def forget-password-doc-container  {:margin-right 16})
(def forget-password-step-container {:flex-direction :row :margin-top 14})
```